packages/expo-updates/ios/EXUpdates/CodeSigning/EXUpdatesCertificateChain.swift

```diff
@@ -179,6 +183,24 @@ extension Array where Element == Certificate {
     try trust.setAnchorCertificates([anchorSecCert])
     try trust.disableNetwork()
     try trust.evaluate()
+    
+    if (count > 1) {
+      let (_, rootX509Cert) = self.last!
+      if (!rootX509Cert.isCACertificate()) {
+        throw EXUpdatesCodeSigningError.CertificateRootNotCA
+      }
+      
+      var lastExpoProjectInformation = try rootX509Cert.expoProjectInformation()
+      // all certificates between root and leaf (non-inclusive)
+      for i in (0...(count - 2)).reversed() {
```

Comment:
Since i=0, don't we (correctly) include the leaf certificate in the check? We want to check every certificate except the root, which has no parent (though one could argue that the root is self-signed and is its own parent, so including the root would be OK too):
```suggestion
      // all certificates between (root, leaf]
      for i in (0...(count - 2)).reversed() {
```

Reply:
Ah, this is the comment from the android code, which is slightly different since we do other things in the loop in android and need to do the leaf check separately. I did mean to include the the leaf in the check here, and just forgot to update the comment.